BREAKING_CHANGES.md

```diff
@@ -117,6 +117,21 @@ New features/fixes include:
   - Use `duration` instead which accepts `ms` defaults to 1250ms
 - `<Skeleton />` loader has renamed `dark` prop to `isDark`
 
+## @zendeskgarden/react-modals
+
+- Boolean props for this package are updated to align with `isBoolean` naming. The following are
+  prop name changes for the `<Modal>` component:
+
+  - `animate` -> `isAnimated`
+  - `center` -> `isCentered`
+  - `large` -> `isLarge`
+
+- `<Close>` no longer accepts the `focused` and `hovered` props. The styling for these states are
+  included with the component, and focusvisible is used.
+
+- The `<Backdrop>` and `<ModalView>` components are deprecated. The abstraction for these
```

Comment:
> deprecated

Implies that they can still consume and use these components until they are removed. We should be able to just drop them.

Reply:
You're right, I've updated this sentence to be more explicit and clear. Thank you!